devicetypes/smartthings/zwave-door-window-sensor.src/zwave-door-window-sensor.groovy

```diff
@@ -92,7 +92,7 @@ private getCommandClassVersions() {
 def parse(String description) {
 	def result = null
 	if (description.startsWith("Err 106")) {
-		if ((zwaveInfo.zw == null && state.sec != 0) || zwaveInfo?.zw?.endsWith("s")) {
+		if ((zwaveInfo.zw == null && state.sec != 0) || zwaveInfo?.zw?.contains("s")) {
```

Comment:
The null checks on `zwaveInfo` are not consistent across all these changes

Reply:
It's not clear to me that `zwaveInfo` can ever be null. Do you have examples where it is?

Reply:
I think these are all coming from very old installed devices that predate it.

Reply:
Based on the `getZwaveInfo` implementation I don't see how it can be null regardless of when the device was installed. I could be misreading the code but that's why I'm asking if you've seen examples where it is null?

Reply:
So `zwaveInfo.zw` can be null. `zwaveInfo` might never be